ipfsconn/ipfshttp/ipfshttp.go

```diff
@@ -806,6 +811,47 @@ func (ipfs *Connector) RepoStat(ctx context.Context) (*api.IPFSRepoStat, error)
 	return &stats, nil
 }
 
+// RepoGC performs a garbage collection sweep on the cluster peer's IPFS repo.
+func (ipfs *Connector) RepoGC(ctx context.Context) (*api.RepoGC, error) {
+	ctx, span := trace.StartSpan(ctx, "ipfsconn/ipfshttp/RepoGC")
+	defer span.End()
+
+	repoGC := api.RepoGC{
+		Keys: make([]api.IPFSRepoGC, 0),
```

Comment:
More idiomatic to do `Keys: []api.IPFSRepoGC{}`

Reply:
Also I see no reasons for it to not be a pointer from here (now doing `&repoGC` in multiple places).